src/mslice/presenters/plot_options_presenter.py

```diff
@@ -27,6 +34,31 @@ def _xy_config_modified(self, key):
         getattr(self, '_xy_config')[key] = getattr(self._view, key)
         self._xy_config['modified'] = True
 
+    def _update_font_sizes_buffer(self):
+        self._default_font_sizes_config = self._model.all_fonts_size.copy()
+
+    def _set_all_plot_fonts(self):
+        new_config_dict = self._default_font_sizes_config.copy()
+
+        fonts_size = self._view.all_fonts_size
+        if fonts_size is not None:
+            new_config_dict = {key: fonts_size for key in new_config_dict}
+
+        self._model.all_fonts_size = new_config_dict
+
+    def _set_font_sizes_tooltip(self):
+        tip = self._convert_font_config_to_tooltip()
+        self._view.allFntSz.setToolTip(tip)
+        self._view.sclUpFntSz.setToolTip(tip)
+        self._view.sclDownFntSz.setToolTip(tip)
```

Comment:
At the top of the scale tooltips, can you put something like "Increment font sizes\n\n" for the up arrow, and "Decrement font sizes\n\n" for the down arrow?